packages/e2e-tests/src/tests/SessionSigs.spec.ts

```diff
@@ -0,0 +1,825 @@
+      jsParams: {
+        dataToSign: alice.loveLetter,
+        publicKey: alice.authMethodOwnedPkp?.publicKey,
+      },
+    });
+  };
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  const res = await Promise.all([fn(1), fn(2), fn(3)]);
+
+  res.forEach((r) => {
+    expect(r?.signatures?.sig.r).toBeDefined();
+    expect(r?.signatures?.sig.s).toBeDefined();
+    expect(r?.signatures?.sig.dataSigned).toBeDefined();
+    expect(r?.signatures?.sig.publicKey).toBeDefined();
+  });
+};
+
+const decryptString = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
+
+  const alice = await devEnv.createRandomPerson();
+  // set access control conditions for encrypting and decrypting
+  const accs = AccessControlConditions.getEmvBasicAccessControlConditions({
+    userAddress:
+      generator.name === 'getEoaSessionSigs'
+        ? alice.wallet.address
+        : alice.authMethodOwnedPkp?.ethAddress!,
+  });
+
+  const litActionSessionSigs = await generator(devEnv, alice);
+
+  const encryptRes = await LitJsSdk.encryptString(
+    {
+      accessControlConditions: accs,
+      dataToEncrypt: 'Hello world',
+    },
+    devEnv.litNodeClient as unknown as ILitNodeClient
+  );
+
+  // -- Expected output:
+  // {
+  //   ciphertext: "pSP1Rq4xdyLBzSghZ3DtTtHp2UL7/z45U2JDOQho/WXjd2ntr4IS8BJfqJ7TC2U4CmktrvbVT3edoXJgFqsE7vy9uNrBUyUSTuUdHLfDVMIgh4a7fqMxsdQdkWZjHign3JOaVBihtOjAF5VthVena28D",
+  //   dataToEncryptHash: "64ec88ca00b268e5ba1a35678a1b5316d212f4f366b2477232534a8aeca37f3c",
+  // }
+
+  // -- assertions
+  expect(encryptRes.ciphertext).toBeDefined();
+
+  expect(encryptRes.dataToEncryptHash).toBeDefined();
+
+  const accsResourceString =
+    await LitAccessControlConditionResource.generateResourceString(
+      accs,
+      encryptRes.dataToEncryptHash
+    );
+
+  const litActionSessionSigs2 = await generator(devEnv, alice, [
+    {
+      resource: new LitAccessControlConditionResource(accsResourceString),
+      ability: LitAbility.AccessControlConditionDecryption,
+    },
+  ]);
+
+  // -- Decrypt the encrypted string
+  const decryptRes = await LitJsSdk.decryptToString(
+    {
+      accessControlConditions: accs,
+      ciphertext: encryptRes.ciphertext,
+      dataToEncryptHash: encryptRes.dataToEncryptHash,
+      sessionSigs: litActionSessionSigs2,
+      chain: 'ethereum',
+    },
+    devEnv.litNodeClient as unknown as ILitNodeClient
+  );
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  expect(decryptRes).toEqual('Hello world');
+};
+
+const broadcastAndCollect = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+) => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
```

Comment:
🔪 ideally this would be replaced with wrapping the test definition function call itself in a conditional check, see above comment.

That being said, do we _really_ never run any test that uses this function on manzano -- and should it be disabled on `datil-test` if it was disabled on `manzano`?